lib/model.js

```diff
@@ -227,7 +228,7 @@ var Events = require('./events'),
                     return Vow.reject(validateResults.reduce(function (err, validationResult, k) {
                         if (validationResult !== true) {
                             if (validationResult === false) {
```

Comment:
Is it possible to return a string value from Attribute.validate method? It can be used as an error message for an `Attribute.ValidationError`.